client/lib/components/recent_numbers_graph.dart

```diff
@@ -109,7 +140,9 @@ class _RecentNumbersBarGraphState extends State<RecentNumbersBarGraph> {
         bars.add(
           BarChartRodData(
             y: 0,
-            color: Constants.textColor.withOpacity(.3),
+            color: Constants.textColor.withOpacity(
+              .3,
+            ),
```

Comment:
Style change above is allowed due to editing code. This section shouldn’t be changed as you’re not changing anything.

Reply:
Originally had a change on that line -- will revert now